templates/autonomous-car.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+    <title>Autonomous Car Assignment</title>
+</head>
+<body>
+    <h3>Autonomous Car Assignment</h3>
+    <div>Build an application / tool in React or Python with the following criteria:</div>
+    <ul>
+        <li>Fetches a car route from this API server</li>
+        <li>Takes a set of instructions as input</li>
+        <li>Determine's if the route was a success or a failure</li>
+        <li>If it was a failure, determine at which step or position that failed</li>
+    </ul>
+    <p>
+        There are different levels to this assignment, level 1 being the easiest, level 2
+        having some gotchas, etc. <br/>If you complete a level with ease, feel free to jump to
+        the next one. <strong>Please timebox your solution to 1 day max</strong>.
+    </p>
+    <h4>Level 1</h4>
+    <ul>
+        <li>Fetch the routes from this server at: http://localhost:5000/autonomous-car/routes/</li>
+    </ul>
+    <h4>Level 2</h4>
+    <ul>
+        <li>Fetch the routes by year: http://localhost:5000/autonomous-car/routes/success|failure|empty...></li>
```

Comment:
is this by year? :)
Also, if you're going to denote options, you should wrap them in square brackets.
`[success|failure|empty|...]`

Reply:
Ha whoopsy again!